app/models/user.rb

```diff
@@ -0,0 +1,3 @@
+class User < ApplicationRecord
+  has_secure_password validations: false
```

Comment:
We won't use AR validations, for what this ones?

Reply:
This is not validation, It's automaticaly creates a hashed password(by bcrypt) and stores it in password digest column.

Reply:
And I've disabled here validation for it